calcom/cal.com
# feat: added ability to assign event to all members

## What does this PR do?



Fixes https://github.com/calcom/cal.com/issues/10991

/claim #10991

Have added the ability to assign all team members a managed event type.



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.
Design Provided :
![265071602-ae6857af-f638-4331-8c3d-eee82807f7c1](https://github.com/calcom/cal.com/assets/47187878/b0fd405f-b019-4f98-a8af-c3b501a1521e)

## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?



- Create a team
- Created a managed event with this team
- Now got to settings -> assignment for this event type
- Inside Assignment check the box above assign to drop down
- See that all the members of the team get assigned to this event type (can check this from the db directly or by logging in ti the team member accounts individually)
- Also test by inviting a new member to the team that they also get this event type assigned as the checkbox mentioned that the event type will be assigned to everyone in the team whenever added.

## Mandatory Tasks


## Checklist



- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
